bin/headers-production.lambda.js

```diff
@@ -1,5 +1,5 @@
 const api = 'https://eapd-api.cms.gov';
```

Comment:
The AWS Lambdas for both Staging and Prod have all of the consts in double quotes.